### ISSUE TITLE
Mapping issues with Kilo that need to be resolved

### ISSUE THREAD

**Issue by miniusAreas** [BeeStation/BeeStation-Hornet#474]
Hi. I'm writing this as a actual mapper who's gone through Kilo, picked it apart, and now making a issue report because I CANT FIND THE PR THAT ADDED IT.

Regardless, if you don't want to listen to all the issues, or simply don't have the time, Kilo is broken beyond feasibility, and as far as I've seen, no one wishes to fix it, nor claim responsibility for maintaining it. Therefor, in typical fashion, it must be removed.

The issues.
- [ ] Random tile below holodeck that disconnects the pipes and wires
- [ ] Missing walls (Library, Janitorial Closet)
- [ ] Tile Decals on Carpet/wood (Bar, Library. These are not ment to be placed on the following.)
- [ ] Hostile Mobs in maint. (No other map has these. We are not Discordia, nor should we be giving assistants a full set of sec armor for welding a spider to death.)
- [ ] Caps Escape Route has a shutter for cockblocking. (Balance issue. You either have a door or a wall, not a shutter and a door.)
- [ ] AI Sat can be one-shot by anyone with a syndicate bomb (No Other AI sat is like this. No AI sat should be like this. Why.)
- [ ] AI Upload Turret can shoot into the bridge (WHO EVEN MADE IT LIKE THIS.)
- [ ] Chicken in Toxins Mix Chamber (THIS IS A SHITTY JOKE AND JUST BLOAT.)
- [ ] Dangerous Atmos Filter tanks have emergency vent buttons (Balance issue. No one should be fucking needing to press these if they play atmos. This just makes it all the more easy to grief or stop someone who's allowed to do it to plasmaflood.)
- [ ] Warden does not have direct access to armory (WHO DESIGNED THIS FUCKING BRIG.)
- [ ] You can look into the toilets from the brig/Virology (W H Y.)
- [ ] Unnecessary pipes/wires through walls (THIS IS A SIN. NEVER DO THIS. FUCK ANYONE WHO DOES THIS.)
- [ ] Rwall blocking Vault enterence 1/2 (Why's it even HAVE two!)
- [ ] Plants growing in space (W H Y.)
- [ ] Heads get a mini-egun (They already get disablers.)
- [ ] Xenobio has a maint door into the coldroom (This is just shitty mapping)
- [ ] Aux Base is hidden in the least-likely to find place (WHY IS IT THERE AND NOT NEAR ARRIVALS / CARGO LIKE EVERY OTHER GODDAMM MAP)
- [ ] Plasmeme EVA suits are just sitting on the floor (Smells like Lazyness.)
- [ ] Caps office has a fake NAD (THIS IS DUMB. IT'S LIKE YOUR BAITING THE CAP TO TAKE IT SO THAT THE REAL NAD SITS IN HIS OFFICE ALL SHIFT SO THAT NUKIES SPAWN AND GREENTEXT INSTANTLY)
- [ ] Bartender gets lethals (THIS BREAKS FUCKING SPACE LAW. WHY.)

And more. Theres probably much much more not listed here. Toxic made a removal PR while I was typing this, bless him. And before you do, Qwerty - This map is done. No one wants to maintain it, I don't want to maintain it, and no one likes it. Time to let it die.

**Comment by MiniMeatwad** [comment]
No kilo station.

**Comment by Toxici11i** [comment]
- [ ] Kilos atmos also doesnt work, plasmaflooding w/o tanks is seemingly impossible, distro doesnt work as it should.
- [ ] Also you cant overpressurise distro for some reason ( i tested this while playing, i was trying to send fuckloads of o2 down the atmos pipes so i could drain it at toxins. Interestingly, there was no pressure change on ANY of the pipes leading out of atmos
In total, this map is fucking gay
- [ ] also toxins gets less TTVs than tanks.
again.

**Comment by pireamaineach** [comment]
Improve don't remove.

**Comment by MarkSuckerberg** [comment]
Can't fix what can't be fixed.

**Comment by pireamaineach** [comment]
I don't see a single complaint in this list that isn't fixable.

**Comment by qwertyquerty** [comment]
Improve don't remove

**Comment by zeskorion** [comment]
well, remove it from the vote until it's improved

**Comment by MarkSuckerberg** [comment]
Put it up to vote then, see what the people who actually play it think

**Comment by pireamaineach** [comment]
Players are braindead, don't let them nake decisions, especially about removals.

**Comment by pireamaineach** [comment]
You want to put it to a vote to justify your removal fetish.

**Comment by MarkSuckerberg** [comment]
Saying players are braindead and that this is only for my removal fetish doesn't negate the fact that players *play* on the maps

**Comment by pireamaineach** [comment]
They play on the maps, but they don't map.

Why would we let players make mapping decisions?

**Comment by pireamaineach** [comment]
Appealing to popularity is a bad argument.

**Comment by MarkSuckerberg** [comment]
I honestly don't understand your argument. The ones who use the map shouldn't have a say on what they maps they play?

**Comment by pireamaineach** [comment]
They do. You're just upset that once in a blue moon they vote for a map you don't like, so you want to remove it.

**Comment by MarkSuckerberg** [comment]
And... so... maybe... call a vote? Let the players decide?

**Comment by zeskorion** [comment]
the thing is, the time the map vote happens, most people are occupied by shuttle bizniz and eorg. in my personal opinion, we should just have the server constantly cycle delta, meta, and pubby

**Comment by MarkSuckerberg** [comment]
That would be ideal

**Comment by Toxici11i** [comment]
> They do. You're just upset that once in a blue moon they vote for a map you don't like, so you want to remove it.

why are you defending an unfinished map that literally on release could game end the station due to one guy aggroing 3-4 mobs
why are you defending an unfinished map that not only has non-functioning atmos, but power issues like the SM not being wired up to the grid? And the SMES also not being wired up to the grid?
why are you defending an unfinished map that literally has hostile mobs like basilisks and ancient goliaths/hivelords outside of the station, which are easy to aggro

**Comment by pireamaineach** [comment]
Or we could just, y'know, let players decide map-to-map like they currently do. But I guess since that occasionally does things you don't like, players voices are irrelevant there.

**Comment by pireamaineach** [comment]
> > They do. You're just upset that once in a blue moon they vote for a map you don't like, so you want to remove it.
> 
> why are you defending an unfinished map that literally on release could game end the station due to one guy aggroing 3-4 mobs
> why are you defending an unfinished map that not only has non-functioning atmos, but power issues like the SM not being wired up to the grid? And the SMES also not being wired up to the grid?
> why are you defending an unfinished map that literally has hostile mobs like basilisks and ancient goliaths/hivelords outside of the station, which are easy to aggro

Because removal doesn't fix these issues and it's not like this map doesn't have potential. The map isn't inherently awful. You're laying out very easily fixable issues. Hell, with such a comprehensive list, why not just fix them?

**Comment by MarkSuckerberg** [comment]
Then let the players decide, and when it embraces its potential it can be readded. In the meantime, remove it from rotation at least. It has fundamental problems. Fix it then everyone will want it back. Although personally I don't think it can or will be fixed.

**Comment by pireamaineach** [comment]
Except if you remove it nobody will care to fix it. I've seen this dozens upon dozens of times. You're the one with the problem, you fix it.

**Comment by MarkSuckerberg** [comment]
Out of our permanent team of about three or four mappers, which includes me, only one has agreed to fix it. I think that's too little for an in-rotation map.

**Comment by zeskorion** [comment]
if it's removed, and no one fixes it, then it was a good thing it was removed

**Comment by pireamaineach** [comment]
So basically you want to remove it because you have problems with it but are too lazy to fix them?

**Comment by pireamaineach** [comment]
If it stays, people who have issues with it, such as yourselves, might finally be motivated to fix them rather than taking the easy way out.

**Comment by MarkSuckerberg** [comment]
Not lazy, just not a fan of the design and I have much bigger fish to fry

**Comment by pireamaineach** [comment]
Seems like a pretty decently sized fish to me.

**Comment by MarkSuckerberg** [comment]
The right course of action would be to decommission it until it's, uh, "fixed." This would be the best solution for all parties

**Comment by pireamaineach** [comment]
The people who have the issues should be expected to fix them. If they're allowed to remove it altogether, they're not going to do shit. It shouldn't fall onto anyone's shoulders but yours.

**Comment by MarkSuckerberg** [comment]
Well then I think it should be removed. There's no shortage of people that don't want kilo on the rotation on and off of the mapping, coding, spriting, or administration team. Donators, too.

**Comment by Kerbin-Fiber** [comment]
Funniest part is, it doesn't matter if it's removed or not. This is all open source, if you really want the map to be in rotation, start fixing it now and make a pr if it's removed, pireamainach

Edit: basically just fix it yourself

**Comment by MarkSuckerberg** [comment]
Problem solved.

**Comment by CrackerJackBeta** [comment]
"Players are braindead" "The people who have the issues should be expected to fix them."
Yeah, the issue is that people don't want Kilo, and they're going to fix that. Also, the unfinished map. 19 + issues should be a sign that something is wrong.

**Comment by Toxici11i** [comment]
> > > They do. You're just upset that once in a blue moon they vote for a map you don't like, so you want to remove it.
> > 
> > 
> > why are you defending an unfinished map that literally on release could game end the station due to one guy aggroing 3-4 mobs
> > why are you defending an unfinished map that not only has non-functioning atmos, but power issues like the SM not being wired up to the grid? And the SMES also not being wired up to the grid?
> > why are you defending an unfinished map that literally has hostile mobs like basilisks and ancient goliaths/hivelords outside of the station, which are easy to aggro
> 
> Because removal doesn't fix these issues and it's not like this map doesn't have potential. The map isn't inherently awful. You're laying out very easily fixable issues. Hell, with such a comprehensive list, why not just fix them?

just because its easily fixable doesnt mean that it shouldve been released in the first place. Unfinished maps have 0 place on a highpop server, it ruins rounds. Also nobody wants to maintain kilo because its a pain in the ass because it breaks MULTIPLE mapping conventions

**Comment by pireamaineach** [comment]
So you want the person who doesn't have a problem with the map to fix issues you have with the map?

Breaking conventions is a positive thing and more reason for the map to remain.

**Comment by Toxici11i** [comment]
no as in they used layered air pipes and disposal pipes which makes it incredibly hard to map and expand or fix issues

**Comment by qwertyquerty** [comment]
Toxics PR was closed because he doesn't know how to disable a map

**Comment by MiniMeatwad** [comment]
If it's so easy to fix, why don't you fix it? @pireamaineach

**Comment by qwertyquerty** [comment]
I've converted all of the noted issues to checkboxes.

**Comment by prophed** [comment]
1. I think we fixed the SM problem? Sorry it was my first map
2. When I was working on kilo I just used the forum post "Official kilostation "fix this shit" thread" and gathered all the complaints and problems from the map. This new list is nutty and everything can be fixable.
3. Sadly when I asked in the DISCORD, where THE PLAYERS PLAY THE MAP,  if there are any other problems with kilo they told me to delete it and kms so I just quit trying to fix the map.
4. Yes, it is an ass map to fix and all pipes need to be checked. 

tl;dr :map is fixable but if 90% of population is telling you to delete it then whats the point of spending hours checking every tile, reason why I dropped it.
P.S. : would be easier to just make a map with a design that everyone enjoys.

**Comment by Kerbin-Fiber** [comment]
I don't think Kilo is coming back

**Comment by yorii** [comment]
Mappers plz

**Comment by prophed** [comment]
This map needs to be reconstructed in the right way. Basically coping the basic design from 0.

**Comment by Crossedfall** [comment]
https://forums.beestation13.com/t/kilo-station-rework-input-thread/5758/17
